static/app/components/core/slider/index.tsx

```diff
@@ -14,200 +21,117 @@ export function Slider({
   return <StyledSlider ref={ref} type="range" {...props} />;
 }
 
+export const Slider = withChonk(LegacySlider, ChonkSlider);
+
 const StyledSlider = styled('input')<React.InputHTMLAttributes<HTMLInputElement>>`
-  ${p => (p.theme.isChonk ? ChonkSliderStyles(p as any) : SliderStyles(p))}
-`;
+  /* stylelint-disable-next-line property-no-vendor-prefix */
+  -webkit-appearance: none;
+  appearance: none;
+  width: 100%;
+  background: transparent;
 
-function SliderStyles(p: {theme: Theme}) {
-  return css`
+  &::-webkit-slider-runnable-track {
+    width: 100%;
+    height: 3px;
+    cursor: pointer;
+    background: ${p => p.theme.border};
+    border-radius: 3px;
+    border: 0;
+  }
+
+  &::-moz-range-track {
+    width: 100%;
+    height: 3px;
+    cursor: pointer;
+    background: ${p => p.theme.border};
+    border-radius: 3px;
+    border: 0;
+  }
+
+  &::-webkit-slider-thumb {
+    box-shadow: 0 0 0 3px ${p => p.theme.background};
+    height: 17px;
+    width: 17px;
+    border-radius: 50%;
+    background: ${p => p.theme.active};
+    cursor: pointer;
     /* stylelint-disable-next-line property-no-vendor-prefix */
     -webkit-appearance: none;
     appearance: none;
-    width: 100%;
-    background: transparent;
+    margin-top: -7px;
+    border: 0;
+    transition:
+      background 0.1s,
+      box-shadow 0.1s;
+  }
+
+  &::-moz-range-thumb {
+    box-shadow: 0 0 0 3px ${p => p.theme.background};
+    height: 17px;
+    width: 17px;
+    border-radius: 50%;
+    background: ${p => p.theme.active};
+    cursor: pointer;
+    /* stylelint-disable-next-line property-no-vendor-prefix */
+    -webkit-appearance: none;
+    appearance: none;
+    margin-top: -7px;
+    border: 0;
+    transition:
+      background 0.1s,
+      box-shadow 0.1s;
+  }
+  &:focus {
```

Comment:
that focus style looks a bit different than our other formFields:

![Screenshot 2025-04-01 at 15 31 08](https://github.com/user-attachments/assets/5d32bada-5255-4d87-804e-efbf1b29ecf3)

vs. e.g. Input:

![Screenshot 2025-04-01 at 15 31 57](https://github.com/user-attachments/assets/4c15b7e2-b3b5-41a0-ab36-a706c05f9f66)

Reply:
Great catch!

Reply:
fixed this here: 40383b64